packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -190,13 +197,20 @@ const SceneView = ({
             false
           : gestureEnabled
       }
+      homeIndicatorHidden={homeIndicatorHidden}
+      navigationBarColor={navigationBarColor}
+      navigationBarHidden={navigationBarHidden}
       replaceAnimation={animationTypeForReplace}
       stackPresentation={presentation === 'card' ? 'push' : presentation}
       stackAnimation={animation}
       screenOrientation={orientation}
       statusBarAnimation={statusBarAnimation}
       statusBarHidden={statusBarHidden}
       statusBarStyle={statusBarStyle}
+      statusBarColor={statusBarColor}
+      statusBarTranslucent={statusBarTranslucent}
+      swipeDirection={swipeDirection}
+      transitionDuration={transitionDuration}
```

Comment:
I think it would be better to call it `animationDuration` here since the main prop is called `animation` so we have all props regarding animating between screens close.

Reply:
So the change would be

`transitionDuration={animationDuration)`, right?

Reply:
Yes